EzPC/EzPC/test_suite_float/binop.ezpc

```diff
@@ -0,0 +1,57 @@
+(*
+
+Authors: Nishant Kumar, Aseem Rastogi.

```

Comment:
Please add your name instead of mine in all these new files!

Reply:
updated with Anwesh's name, test cases are written by @anuwu